api/repositories/route_repository.go

```diff
@@ -103,7 +103,7 @@ type DeleteRouteMessage struct {
 	SpaceGUID string
 }
 
-func (m CreateRouteMessage) toCFRoute() networkingv1alpha1.CFRoute {
+func (m CreateRouteMessage) toCFRoute(domainNamespace string) networkingv1alpha1.CFRoute {
```

Comment:
Unfortunately it doesn't fit neatly into our data structures. The payload is unmarshalled into a `payloads.RouteCreate` struct. If this is part of the information of the Domain, then it should live in `RouteCreate.Relationships.Domain.Data`, which is of type `Relationship` and only has a GUID field.

Potentially, it could be assigned to a field on `CreateRouteMessage` after `payloads.RouteCreate.ToMessage()` is invoked. This might just hide it in a different place, however.

Reply:
After some discussion with AMER folks:

We decided to move the namespace into the `payloads.RouteCreate.ToMessage` function as an argument that is assigned to a field on the `CreateRouteMessage` struct. Currently applying changes to update the PR.